src/AWS.Deploy.Common/DeploymentManifest/DeploymentManifestEngine.cs

```diff
@@ -57,7 +57,14 @@ public async Task UpdateDeploymentManifestFile(string saveCdkDirectoryFullPath,
                 if (_fileManager.Exists(deploymentManifestFilePath))
                 {
                     deploymentManifestModel = await ReadManifestFile(deploymentManifestFilePath);
-                    deploymentManifestModel.DeploymentManifestEntries.Add(new DeploymentManifestEntry(saveCdkDirectoryRelativePath));
+                    if (deploymentManifestModel.DeploymentProjects == null)
+                    {
+                        deploymentManifestModel.DeploymentProjects = new List<DeploymentManifestEntry> { new DeploymentManifestEntry(saveCdkDirectoryRelativePath) };
+                    }
```

Comment:
This and L100 check look redundant, `DeploymentManifestModel` has constructor which always initializes with a non-null List<DeploymentManifestEntry>.

Reply:
They are not redundant. When we are deserializing them, if they don't exist in the file, they will be null. Ive ran into null exceptions multiple times which is why i added the check.

Reply:
That's interesting, there must be some config that we are missing here. We can check it later.